spec.md

```diff
@@ -1399,6 +1500,127 @@ Errors](#service-broker-errors).
+endpoint returns `"state": "succeeded"` for a [Provisioning](#provisioning)
+operation.
+Otherwise, platforms SHOULD NOT attempt to call this endpoint under any
+circumstances.
+
+### Request
+
+##### Route
+`GET /v2/service_instances/:instance_id`
+
+`:instance_id` is the identifier of a previously provisioned instance.
+
+##### cURL
+```
+$ curl 'http://username:password@broker-url/v2/service_instances/:instance_id' -X GET -H "X-Broker-API-Version: 2.11"
+```
+
+### Response
+
+| Status Code | Description |
+| --- | --- |
+| 200 OK | The expected response body is below. |
+| 404 Not Found | MUST be returned if the service instance does not exist or if a provisioning operation is still in progress. The expected response body is `{}`. |
+| 422 Unprocessable Entity | MUST be returned if the service instance is being updated and therefore cannot be fetched at this time. |
+
+Responses with any other status code will be interpreted as a failure.
+Brokers can include a user-facing message in the `description` field; for
+details see [Broker Errors](#service-broker-errors).
+
+##### Body
+
+For success responses, a broker MUST return the following fields. For error
+responses, see [Broker Errors](#service-broker-errors).
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| dashboard_url | string | The URL of a web-based management user interface for the service instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`9189kdfsk0vfnku` in the example below). Note: a broker that wishes to return `dashboard_url` for a service instance MUST return it with the initial response to the provision request, even if the service is provisioned asynchronously. |
+| parameters | JSON object | Configuration options for the service instance. Service brokers SHOULD ensure that the configuration parameters match the service instance update schema they provide to platforms via the [Catalog](#catalog-management) endpoint (this will enable platforms to offer additional features such as pre-populated form fields in UIs). If brokers cannot fetch all configuration parameters, they MUST NOT return any. |
+
+\* Fields with an asterisk are REQUIRED.
+
+```
+{
+  "dashboard_url": "http://example-dashboard.example.com/9189kdfsk0vfnku",
+  "parameters": {
+    "billing-account": "abcde12345"
+  }
+}
+```
+
+## Fetching a Service Binding
+
+If `"binding_retrievable" :true` is declared for a service in the
+[Catalog](#catalog-management) endpoint, brokers MUST support this endpoint for
+all services and plans that support bindings (`"bindable": true`) and this
+endpoint MUST be available immediately after the
+[Polling Last Operation for Service Bindings](#polling-last-operation-for-service-bindings)
+endpoint returns `"state": "succeeded"` for a [Binding](#binding) operation.
+Otherwise, platforms SHOULD NOT attempt to call this endpoint under any
```

Comment:
Do we need the last sentence? Wouldn't they just get back a 404 and no harm is done?  Same question for GET on instances too I guess.

Reply:
re #159